lib/terraforming/resource/iam_instance_profile.rb

```diff
@@ -16,23 +16,23 @@ def initialize(client)
       end
 
       def tf
-        apply_template(@client, "tf/iam_instance_profile")
+        apply_template(@client, 'tf/iam_instance_profile')
       end
 
       def tfstate
         iam_instance_profiles.inject({}) do |resources, profile|
           attributes = {
-            "arn" => profile.arn,
-            "id" => profile.instance_profile_name,
-            "name" => profile.instance_profile_name,
-            "path" => profile.path,
-            "roles.#" => profile.roles.length.to_s,
+            'arn' => profile.arn,
+            'id' => profile.instance_profile_name,
+            'name' => profile.instance_profile_name,
+            'path' => profile.path,
+            'roles.#' => profile.roles.length.to_s,
```

Comment:
__[rubocop]__
* Avoid comma after the last item of a hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)